src/server/src/server_main.cpp

```diff
@@ -0,0 +1,60 @@
+//
+// Created by kevin on 6/25/22.
+//
+#include <ros/ros.h>
+#include <client/current_time.h>
+#include <client/login.h>
+
+ros::NodeHandle *server_node_handle_pointer = NULL;
+ros::Subscriber server_subscriber;
+ros::ServiceServer service_server;
+
+void time_callback(const client::current_time::ConstPtr &msg) {
+    ROS_INFO("Topic received from : %s , Whose time is %d/%d/%d-%d:%d:%d ."
+            , (msg->name).c_str()
+            , msg->year
+            , msg->month
+            , msg->day
+            , msg->hour
+            , msg->minute
+            , msg->second
+    ); //���
+}
+
+bool login_handle(client::login::Request &req, client::login::Response &ack) {
+    if(req.req_code == 1) {
+        ROS_INFO("A client: %s has logged in!", req.node_name.c_str());
+        ack.ack_code = 10;
+        server_subscriber = (*server_node_handle_pointer).subscribe(req.node_name, 1, time_callback);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]